pdns/recursordist/pdns_recursor.cc

```diff
@@ -2666,7 +2658,7 @@ void makeUDPServerSockets(deferredAdd_t& deferredAdds, Logr::log_t log)
       if (address.sin6.sin6_family == AF_INET6 && setsockopt(socketFd, IPPROTO_IPV6, IPV6_V6ONLY, &one, sizeof(one)) < 0) {
         int err = errno;
         SLOG(g_log << Logger::Warning << "Failed to set IPv6 socket to IPv6 only, continuing anyhow: " << stringerror(err) << endl,
-             log->error(Logr::Warning, "Failed to set IPv6 socket to IPv6 only, continuing anyhow"));
+             log->error(Logr::Warning, err, "Failed to set IPv6 socket to IPv6 only, continuing anyhow"));
```

Comment:
is this meant to just log the errno number?

Reply:
`void Logger::error(Logr::Priority p, int err, const std::string& msg) const` translates `err` using `strerror(err)`

Reply:
ah, wonderful, I was hoping for something like that